src/actions/getDatasetItems.js

```diff
@@ -1,33 +1,30 @@
-import * as apifyHelper from '../helpers/apifyHelper';
-import path from 'path';
-import parseCsvPromised from '../helpers/csvHelpers';
-import { createFilePromised, createFolderPromised } from '../helpers/fsHelper';
-import command from '../helpers/cliHelper';
-import { DEFAULT_TABLES_OUT_DIR } from '../constants';
+const apifyHelper = require('../helpers/apifyHelper');
+const path = require('path');
+const parseCsvPromised = require('../helpers/csvHelpers');
+const { createFilePromised, createFolderPromised } = require('../helpers/fsHelper');
+const { DEFAULT_TABLES_OUT_DIR, DATA_DIR, DATASET_FILE_NAME } = require('../constants');
 
 const RESULTS_FILE_LIMIT = 50000;
-const DEFAULT_PAGINATION_LIMIT = 1000;
+const DEFAULT_PAGINATION_LIMIT = 5000;
 
 
 /**
  * Outputs all data from Apify datasets to data/out
  */
-export default async function getDatasetItems(apifyClient, maybeDatasetId) {
+module.exports = async function getDatasetItems(apifyClient, maybeDatasetId) {
     const apifyDatasets = await apifyClient.datasets;
     let dataset = await apifyDatasets.getDataset({ datasetId: maybeDatasetId });
     if (!dataset) {
         // Try to find by name
         const datasets = await apifyDatasets.listDatasets({ limit: 99999 });
```

Comment:
API maximum is 1000

Reply:
We need to allow fetching dataset by name at API! This is not a good way. It's easy change.

Reply:
it isn't good like that, we should enable it.
now we can set datasetId with username like 'drobnikj~my-dataset' and it works, but it doesn't work, when user doesn't have username